### PR TITLE
Add check for obsolete GitHub/GitLab URIs

### PR DESCRIPTION
**Pull request by mgorny**

### CODE REVIEW

File: src/pkgcheck/checks/codingstyle.py [hunk 1]
```diff
@@ -302,3 +302,60 @@ def feed(self, entry):
...
+        super().__init__(options)
+        self.regexes = []
+        for regexp, repl in self.REGEXPS:
+            self.regexes.append((re.compile(regexp), repl))
+
+    def feed(self, entry):
+        pkg, lines = entry
+        links = defaultdict(list)
+
+        for lineno, line in enumerate(lines):
```
Comment (radhermit) [inline src/pkgcheck/checks/codingstyle.py]: You'll probably want enumerate(lines, 1) here so lineno isn't off by one, but I can probably fix that on merge.

Comment (mgorny) [reply src/pkgcheck/checks/codingstyle.py]: Done.

**Comment by mgorny** [comment]
Now includes tests as well.

**Comment by radhermit** [comment]
Thanks, merged with some minor changes.

**Pull request closed without merging**